### PR TITLE
Nav Bar and Intro Finished 

### PR DESCRIPTION
**Pull request by Joshsilas**
Nav bar and the intro Page is finished

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,38 +10,12 @@
...
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			
-			
+		
+			<section>
```
Comment (CMeier556) [inline index.html]: Maybe give your sections a class?

File: style.css [hunk 2]
```diff
@@ -15,13 +17,35 @@ header{
...
+	display: flex;
+	justify-content: center;
+	flex-wrap: wrap;
+	margin:0px;
+	align-items: center;
+	gap:0px;
+	
+}
+p.little-intro{
+color: white;
```
Comment (CMeier556) [inline style.css]: Indentation

File: style.css [hunk 3]
```diff
@@ -15,13 +17,35 @@ header{
...
 }
 
 .nav {
 	font-size: 1.5rem;
 	color: white;
 	display: flex;
 	gap:40px;
-	text-decoration: none;
-		
+	text-decoration: none;
```
Comment (CMeier556) [inline style.css:29]: Is this code necessary?

Comment (Joshsilas) [inline style.css]: Yes

File: style.css (hunk 2 above)
Comment (Joshsilas) [reply style.css]: Done

File: index.html [hunk 4]
```diff
@@ -10,38 +10,12 @@
...
+				<p class="intro">HEELLL</p>
+				<p class="intro">OOOOO</p>
+				<p class="little-intro">I am Josh</p>		
+		</section>
```
Comment (CMeier556) [inline index.html]: Closing html tag and head tag on top is not closed

Comment (Joshsilas) [reply index.html]: Thankyou

**Review by CMeier556 — APPROVED** [review]

**Pull request closed without merging**